src/pages/layer-2.js

```diff
@@ -0,0 +1,1097 @@
+                </p>,
+                <p>
+                  <b>The blockchain</b> itself and the history of transaction
+                  data
+                </p>,
+                <p>
+                  <b>The consensus mechanism</b> for the network
+                </p>,
+              ]}
+            />
+            <p>
+              Still confused on Ethereum?{" "}
+              <Link to="/what-is-ethereum/">Learn what Ethereum is</Link>.
+            </p>
+          </Flex50>
+        </TwoColumnContent>
+      </LightGrayContent>
+
+      <PaddedContent>
+        <TwoColumnContent>
+          <FlexContainer
+            flexPercent="50"
+            style={{
+              display: "flex",
+              alignContent: "center",
+              justifyContent: "center",
+            }}
+          >
+            <GatsbyImage
+              image={getImage(data.dao)}
+              style={{ width: "100%" }}
+              objectFit="contain"
+            />
+          </FlexContainer>
+          <FlexContainer flexPercent="50">
+            <h2>Why do we need layer 2?</h2>
+            <p>
+              The three desirable properties of a blockchain are that it is
+              <b>decentralized, secure, and scalable</b>. The{" "}
+              <Link to="https://www.ledger.com/academy/what-is-the-blockchain-trilemma">
+                blockchain trilemma
+              </Link>{" "}
+              states that a simple blockchain architecture can only achieve two
+              out of three. Want a secure and decentralized blockchain? You need
+              to sacrifice scalability. This is where layer 2 networks come in.
+            </p>
+            <p>
+              Ethereum has reached the network's current capacity with{" "}
+              <Link to="https://etherscan.io/chart/tx">
+                1+ million transactions per day
+              </Link>
+              , with high demand for each of these transactions. The success of
+              Ethereum and the demand to use it has caused gas prices to rise
+              substantially. Therefore the{" "}
+              <Link to="/developers/docs/scaling/">
+                need for scaling solutions
+              </Link>{" "}
+              has increased in demand as well.
+            </p>
+
+            <h3>Scalability</h3>
+            <p>
+              The main goal of scalability is to increase transaction speed
+              (faster finality), and transaction throughput (high transactions
+              per second), without sacrificing decentralization or security
+              (more on the <Link to="/upgrades/vision/  ">Ethereum vision</Link>
+              ).
+            </p>
+            <p>
+              The Ethereum community has taken a strong stance that it would not
+              throw out decentralization or security in order to scale. Until
+              sharding, Ethereum Mainnet (layer 1) will only be able to process{" "}
```

Comment:
If we need to explain Mainnet === L1 here then we probably haven't done a good enough job explaining it in the Layer 1 section.

```suggestion
              throw out decentralization or security in order to scale. Until
              sharding, Ethereum Mainnet will only be able to process{" "}
```